### PR TITLE
[ZT] Move Gateway applications page up a level (PCX-3453)

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/policies/filtering/application-app-types.md [hunk 1]
```diff
@@ -34,7 +32,7 @@ Using these two selectors in the HTTP policy builder, you can have more granular
 
 ### Applications
 
-A full list of supported applications and their respective app types are available to download [here](/cloudflare-one/static/documentation/applications.csv).
+When you [create a policy for applications](#creating-policies-with-applications-and-app-types) from the Zero Trust dashboard, the **Value** drop-down menu lists all supported applications and their respective app types. To view an up-to-date list outside of the UI, please refer to the [Gateway API guide](https://api.cloudflare.com/#zero-trust-gateway-application-and-application-type-mappings-properties).
```
Comment (marciocloudflare) [inline content/cloudflare-one/policies/filtering/application-app-types.md]:
```suggestion
When you [create a policy for applications](#creating-policies-with-applications-and-app-types) from the Zero Trust dashboard, the _Value_ drop-down menu lists all supported applications and their respective app types. To view an up-to-date list outside of the UI, please refer to the [Gateway API guide](https://api.cloudflare.com/#zero-trust-gateway-application-and-application-type-mappings-properties).
```

Comment (marciocloudflare) [reply content/cloudflare-one/policies/filtering/application-app-types.md]: If this is a drop-down menu, it should be italic

File: content/cloudflare-one/policies/filtering/application-app-types.md [hunk 2]
```diff
@@ -68,9 +66,9 @@ Some applications are incompatible with TLS decryption for a variety of reasons,
...
+Gateway automatically groups applications incompatible with TLS decryption into the _Do Not Inspect_ app type. To ensure that traffic gets through to these applications, you can [create an HTTP policy](#creating-policies-with-applications-and-app-types), select _Application_ as a **Selector**, _in_ as an **Operator**, and check the _Do Not Inspect_ app type in the **Value** field. Then, set the HTTP policy **Action** to _Do Not Inspect_.
 
-Gateway periodically updates the _Do Not Inspect_ app type to include new applications. By creating this _Do Not Inspect_ policy and selecting all applications within the _Do Not Inspect_ app type, you'll ensure that your policy will apply to any new applications that will be added to the app type.
+Gateway periodically updates the _Do Not Inspect_ app type to include new applications. By creating this _Do Not Inspect_ HTTP policy and selecting all applications within the _Do Not Inspect_ app type, you'll ensure that your _Do Not Inspect_ policy will apply to any new applications added to the app type.
```
Comment (marciocloudflare) [inline content/cloudflare-one/policies/filtering/application-app-types.md]:
```suggestion
Gateway periodically updates the _Do Not Inspect_ app type to include new applications. By creating this _Do Not Inspect_ HTTP policy and selecting all applications within the _Do Not Inspect_ app type, you will ensure that your _Do Not Inspect_ policy will apply to any new applications added to the app type.
```

Comment (marciocloudflare) [reply content/cloudflare-one/policies/filtering/application-app-types.md]: We don't use contractions in dev docs

**Review by marciocloudflare — CHANGES_REQUESTED** [review]

File: content/cloudflare-one/policies/filtering/application-app-types.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/policies/filtering/application-app-types.md]: Got it. I was copying the formatting from another part of the document, so I'll go ahead and change those as well.

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**